mysql-test/suite/tianmu/std_data/issue1263-2.txt

```diff
@@ -0,0 +1,2 @@
+"765893","37","27","44221","2021-06-28 02:01:16","计算售罄率出错：ORA-04036: PGA memory used by the instance exceeds PGA_AGGREGATE_LIMITORA-06512: at ""BOSNDS3.O2O_STORESALERATE_SET"", line 49
```

Comment:
what will happens?  """ xxx """, and """" xxxx """"

Reply:
add these test case . Please check the MTR result file or datafile(issue1263-2.txt and issue1263-3.txt). 
1. double enclosed char  will be interpred as a normal char;
2. singe enclosed char before field-terminated string will be interpred as enclose char
3. sing enclosed char with non-filed-termniated string will be interpreted as a normal char;

Reply:
PTAL, @RingsC